plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/nexus/ldap/internal/StaticSecurityConfigurationResourceImpl.groovy

```diff
@@ -67,6 +71,7 @@ class StaticSecurityConfigurationResourceImpl
             ),
             new CPrivilege(
                 id: 'ldap-delete',
+                description: 'Delete permission for Ldap',
```

Comment:
As an acronym should these be "LDAP"?  I am not sure if that's consistant across the site but I feel like it is mostly.